source/examples/xgboost-dask-databricks/notebook.ipynb

```diff
@@ -0,0 +1,2163 @@
+{
```

Comment:
<p>Why is this commented? Should we just remove it if we don't want to run it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>